lib/package-manager.coffee

```diff
@@ -384,7 +384,13 @@ class PackageManager
   emitPackageEvent: (eventName, pack, error) ->
     theme = pack.theme ? pack.metadata?.theme
     eventName = if theme then "theme-#{eventName}" else "package-#{eventName}"
-    @emit eventName, pack, error
+    @emitter.emit eventName, pack, error
+
+  on: (selectors, callback) ->
+    subscriptions = new CompositeDisposable
+    selectors.split(" ").forEach (selector) =>
+      subscriptions.add @emitter.on selector, callback
+    subscriptions
```

Comment:
I fear things [like this](https://github.com/atom/settings-view/blob/37aad1d2b84b95ca83e5f9b43f10fd282ecf9fcb/lib/package-card.coffee#L351-L371) would become _really_ difficult to implement.

I'm going to change this to use `onXXX` methods, but it really smells to me :/